stable/atlantis/README.md

```diff
@@ -51,6 +51,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `podTemplate.annotations` | Specifies additional annotations to use for the StatefulSet | n/a                  |
 | `logLevel`                                  | The level to use for logging.                             | n/a |
 | `orgWhiteList`                              | A whitelist of repositories from which Atlantis will accept webhooks. **This value must be changed for Atlantis to function correctly.**  Accepts wildcard characters (`*`).  Multiple values may be comma-separated.                                                                                                                                      | `github.com/yourorg/*` |
+| `requireApproval` | Whether to enable [apply requirements](https://www.runatlantis.io/docs/apply-requirements.html) globally. | `false` |
```

Comment:
This should be clear that it's talking about the approval apply requirement. There's another apply requirement: mergeable. It would be nice if this PR also added that option `requireMergeable` but it's okay if you don't have time for that.

```suggestion
| `requireApproval` | Whether to require pull request approval prior to applies. See [Approved Requirement](https://www.runatlantis.io/docs/apply-requirements.html#approved). | `false` |
```